src/components/Heading/Heading.test.jsx

```diff
@@ -52,28 +52,28 @@ describe('Heading', () => {
   headingComponentLevels.forEach((headingComponent, index) => {
     test(`Matches the snapshot for heading ${HEADING_LEVELS[index]}`, () => {
       const { asFragment } = render(headingComponent);
-      expect(asFragment(headingComponent)).toMatchSnapshot();
+      expect(asFragment()).toMatchSnapshot();
     });
   });
 
   headingComponentSizes.forEach((headingComponent, index) => {
     test(`Matches the snapshot for size ${PALMETTO_FONT_SIZE_OPTIONS[index]}`, () => {
       const { asFragment } = render(headingComponent);
-      expect(asFragment(headingComponent)).toMatchSnapshot();
+      expect(asFragment()).toMatchSnapshot();
     });
   });
 
   headingComponentSizeOverrides.forEach((headingComponent, index) => {
     test(`Matches the snapshot for heading ${HEADING_LEVELS[index]} and font size ${PALMETTO_FONT_SIZE_OPTIONS[index]}`, () => { // eslint-disable-line max-len
       const { asFragment } = render(headingComponent);
-      expect(asFragment(headingComponent)).toMatchSnapshot();
+      expect(asFragment()).toMatchSnapshot();
     });
   });
 
   headingComponentColors.forEach((headingComponent, index) => {
     test(`Matches the snapshot for color ${PALMETTO_BRAND_COLOR_OPTIONS[index]}`, () => {
       const { asFragment } = render(headingComponent);
-      expect(asFragment(headingComponent)).toMatchSnapshot();
+      expect(asFragment()).toMatchSnapshot();
     });
   });
 });
```

Comment:
unit test instead. if you want to do that in a separate pr, that works too.

Reply:
@nathanyoung I will do this on a separate PR.